### ISSUE TITLE
 [Adaptive Card iOS - Input.text.json]: 'Default text input' label is not associated with its edit box.

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards#8411]
### Target Platforms

iOS

### SDK Version

1.0 (2.4.0-beta.23.1.8.1)

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
Device: iPhone 11
iOS: 16.3.1
Application: Adaptive Cards
App Version: 1.0 (2.4.0-beta.23.1.8.1)
Screen Reader: Voice Over

**Repro Steps:**

1. Launch Adaptive Cards iOS application.
2. Home screen will appear.
3. Activate Voice over
4. Navigate to "V1.0" control and select it.
5. Navigate to 'Elements' control and activate it.
6. Navigate to 'Input.text.json' and activate it.
7. Navigate to the 'Edit field' under 'Default text input' and observe the issue.

**Actual Result:**
'Default text input' label is not associated with its edit box as while navigating to edit box screen reader does convey the information separately.

**Expected Result:**
Label should be associated with its edit box, and it should announce as **'Default text input enter comment double tap to edit.'**.

**Note:**
1. v1.0 - Scenarios - Inputform.json - Issue is repro with textfield as label is not associated.

**User Experience:**
It will impact the screen reader user if the label is not associated with its edit box.
 
**WCAG Reference:**
https://www.w3.org/WAI/WCAG21/Understanding/labels-or-instructions.html

"Have feedback to share on Bugs ? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-apoorva%40microsoft.com%7c551850210d444be9b28308d9691e3dde%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656400998658942%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=l6w0SvVcMSvyGnJ6%2B6uSSgd3braF2dZ7/AdBrqqzIR8%3D&reserved=0) “

### Screenshots

https://user-images.githubusercontent.com/95913748/227923018-63a1ac6e-778a-4472-a571-8aad47987b20.MP4

### Card JSON

```JSON
NA
```
